### DISCUSSION TITLE
An official docker container?

### DISCUSSION THREAD

**Discussion by adyanth** [jesec/flood#378] [Ideas]
This is a very good tool to use in with docker. An official port available on hub.docker.com would increase accessibility.

There are themes for Flood on theme-park here https://docs.theme-park.dev/themes/flood/ an option to apply them to the container would be good too.

Here is what I use now: https://git.adyanth.site/DockerComposeApps/flood-transmission/src/branch/main/floodui/Dockerfile
With the compose file including transmission: https://git.adyanth.site/DockerComposeApps/flood-transmission/src/branch/main/docker-compose.yml

**Comment by jesec** [comment]
…

https://hub.docker.com/r/jesec/flood

**Reply by adyanth** [reply]
Well, for some reason, that did not come up when I was searching for it :)

Thanks!
